pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -34,6 +36,8 @@ const (
 	ReplaceConcurrent ConcurrencyPolicy = "Replace"
 )
 
+const annotationKeyLatestSchedule = workflow.CronWorkflowFullName + "/last-used-schedule"
```

Comment:
Moving the label here was necessary to avoid import cycles. I believe this is okay because we have full accessor methods and the label is private... this way implementation of the "last used schedule" concept is completely abstracted.

Reply:
maybe `previous-schedule`?

Reply:
I think `last-used-schedule` is more accurate. We're interested in the schedule that was last used to execute a workflow. This may or may not be the previous schedule.